### ISSUE TITLE
Running title should change when further reading or exercices are given

### ISSUE THREAD

**Issue by stefan11** [langsci/latex#82]
![bildschirmfoto 2018-03-10 um 15 12 15](https://user-images.githubusercontent.com/4655859/37243162-7180743e-2475-11e8-97b5-2097d21ed7e4.png)

We are not in "Summary and classification" anymore.

**Comment by timmli** [comment]
@stefan11 Could you provide an MWE?

**Comment by kopeckyf** [comment]
@stefan11 In your example, would you like the recto title change to "Further reading" or to a more general title? 
(My proposal would be to integrate your \[furtherreading](https://github.com/langsci/25/blob/e1673bad470b6c0de20dd89eae4c94ba28da2cfc/localcommands.tex#L274) into `langscibook` and add a section mark from the `scrbook` class

**Comment by kopeckyf** [comment]
Yes, there probably is no one size fits all solution. Users who *really* need to change this should use `\rohead{Further reading}` from `scrbook`.
